run_build.sh

```diff
@@ -27,33 +49,49 @@ declare -a PRECIOUS_FILES=("bin/daos"
                            "ior${MPI_SUFFIX}/bin/${MDTEST_BIN}"
                            )
 
-# List of development or test branches to be merged on top of DAOS
-# master branch
-declare -a DAOS_PATCHES=("origin/tanabarr/control-no-ipmctl-May2020"
-                         "origin/mjmac/io500-202104"
-                         )
-
-# List of development or test branches to be merged on top of DAOS
-# release/1.2 branch
-#declare -a DAOS_PATCHES=(
-#                         "origin/mjmac/io500-frontera"
-#                         )
+# Extra branches needed for DAOS to work on frontera
+if [ "${DAOS_BRANCH}" == "release/1.2" ]; then
+    # Need to use the older branch
+    declare -a DAOS_PATCHES=(
+                             "origin/mjmac/io500-frontera"
+                             )
+else
+    # Use the newer branch
+    declare -a DAOS_PATCHES=(
+                             #"origin/mjmac/io500-202104"
+                             "origin/dbohninx-io500-20210601"
```

Comment:
origin/mjmac/io500-202104 is still needed for some older commits, but origin/dbohninx-io500-20210601 works on the tip of master